deploy/roles/eod7/tasks/main.yaml

```diff
@@ -0,0 +1,480 @@
+  template:
+    src: gdm.schemas.j2
+    dest: /etc/gdm/gdm.schemas
+    mode: 0644
+  notify:
+    - restart eod
+  tags:
+  - eod
+
+- name: Remove gpk-update-icon file from /etc/xdg/autostart directory
+  file:
+    path: /etc/xdg/autostart/gpk-update-icon.desktop
+    state: absent
+  tags:
+    - eod
+
+- name: run gdm
+# command: start prefdm  -- for CentOS/RHEL6
+#  command: systemctl start display-manager
+  command: systemctl start gdm
+  register: gdm_status
+  failed_when: gdm_status.rc != 0 and "Job is already running" not in gdm_status.stderr
+  changed_when: gdm_status.rc == 0
+  tags:
+    - eod
+
+# copy eodpatch expect script and execute
+#- name: copy eodpatch.exp to remote
+#  file:
+#    path: /files/eodpatch.exp
+#    dest: /tmp/eodpatch.exp
+#    mode: a+x
+#  tags:
+#    - eod
+    
+#- name: execute eodpatch.exp script
+#  command: /tmp/eodpatch.exp
+#  shell: /tmp/eodpatch.exp
+#  ignore_errors: yes
+#  register: eodpatch
+#  tags:
+#    - eod
+    
+#- name: clear host errors`
+#  meta: clear_host_errors
+    
+#- debug: var=eodpatch
+- name: Run eodpatch expect inline - required before running /usr/local/eod/bin/otecs ...
+  shell: |
+    set timeout 120
+    spawn /usr/local/eod/bin/patch
+
+    match_max 100000
+    expect -exact "\r
+    patch - Patch script for Exceed Connection Server 13.8\r
+    \r
+    \r
+    Proceed to patch permissions bits for /usr/local/eod/bin/sys directory and /usr/local/eod/bin/sys/ewebhost file?\r
+    \[y\]es or \[n\]o, \[Enter\] for no: "
+    send -- "y\r"
+    #expect eof
+    #expect eod
+    exit 0
+  args:
+    executable: /bin/expect
+  tags:
+    - eod
+
+### Ensure EOD starts on boot
+
+- name: Check if EOD is configured to run on boot
+  shell: "{{ eod_install_directory }}/bin/otecs bootstatus"
+  register: eod_bootstatus
+  changed_when: false
+  when: custom_repo
+  tags:
+    - eod
+
+- name: Configure EOD to start on boot
+  shell: "{{ eod_install_directory }}/bin/otecs bootstart"
+  changed_when: eod_bootstatus.stdout == 'off'
+  when: custom_repo
+  tags:
+    - eod
+
+### Allow the admin group to read logs and configs
+
+- name: Ensure that EOD log directory exists
+  file:
+    path: "{{ eod_install_directory }}/log"
+    state: directory
+    owner: root
+    group: root
+    mode: 0640
+  tags:
+    - eod
+
+#- name: Set recursive admin permissions on log file
+#  acl:
+#    path: "{{ eod_install_directory }}"
+#    entity: "{{ restricted_group_name }}"
+#    etype: group
+#    permissions: rwX
+#    recursive: yes
+#    state: present
+#  tags:
+#    - eod
+
+#- name: Set default admin permissions on log file
+#  acl:
+#    path: "{{ eod_install_directory }}"
+#    entity: "{{ restricted_group_name }}"
+#    etype: group
+#    permissions: rwX
+#    default: yes
+#    state: present
+#  tags:
+#    - eod
+
+### Ensure the needed ports are open
+- name: create custom chain
+  command: "iptables -N {{ iptables_chain }}"
+  ignore_errors: yes
```

Comment:
@lel99999 This should only be done in local development, not in our environment as iptables is managed by Puppet.

Reply:
Added conditional: ENV == "DEV.local", now to determine variable injection point.